src/VirtoCommerce.ExperienceApiModule.Core/Helpers/QueryArgumentsHelper.cs

```diff
@@ -0,0 +1,23 @@
+using GraphQL.Types;
+
+namespace VirtoCommerce.ExperienceApiModule.Core.Helpers
+{
+    public static class QueryArgumentsHelper
+    {
+        public static QueryArguments GetComplexTypeQueryArguments<TGraphType>(string name) where TGraphType : IGraphType
```

Comment:
Is it possible to use more specific constraints to achieve method `GetQueryArguments` has two overloads?